### ISSUE TITLE
Uyuni support for windows patch

### ISSUE THREAD

**Issue by maroofaurionpro** [uyuni-project/uyuni#7583]
# Question
Can we also create Centralized windows repository with uyuni tool is there any possibility to do this ?

**Comment by mcalmer** [comment]
No, windows is not supported. Only Linux repositories which provide RPM or DEB packages
